source/_docs/scripts.markdown

```diff
@@ -98,7 +98,7 @@ wait_template: "{{ is_state(trigger.entity_id, 'on') }}"
 ```
 {% endraw %}
 
-It is also possible to use dummy variables, e.g. in scripts, when using `wait_template`.
+It is also possible to use dummy variables, e.g., in scripts, when using `wait_template`.
```

Comment:
Check out [Link](https://jakubmarian.com/comma-after-i-e-and-e-g/). Both versions are possible. I just tend to like the British one better.

Reply:
It OK to like the British one better, but we have a policy on having American English only.